samples/drivers/memc/src/main.c

```diff
@@ -50,7 +53,7 @@ int main(void)
 		memc_write_buffer[i] = (uint8_t)i;
 	}
 	printk("Writing to memory region with base 0x%0x, size 0x%0x\n\n",
-		MEMC_BASE, MEMC_SIZE);
+		((uint32_t) MEMC_BASE), MEMC_SIZE);
```

Comment:
This truncates the MEMC_BASE or promotes it up? Because in first case it may give incorrect info.